### ISSUE TITLE
Android Basics: Get data from the Internet

### ISSUE THREAD

**Issue by intSpartan** [google-developer-training/android-basics-kotlin-mars-photos-app#146]
---
name: Android Basics Unit 4 issue
about: Report problems with the Android Basics Unit 4 codelabs
title: 'Android Basics Unit 4 issue:'
labels: ''
assignees: ''

---

**URL of codelab**
https://developer.android.com/codelabs/basic-android-kotlin-training-getting-data-internet?continue=https%3A%2F%2Fdeveloper.android.com%2Fcourses%2Fpathways%2Fandroid-basics-kotlin-unit-4-pathway-2%23codelab-https%3A%2F%2Fdeveloper.android.com%2Fcodelabs%2Fbasic-android-kotlin-training-getting-data-internet#2

**In which task and step of the codelab can this issue be found?**
Spelling wrong of one of the heading

**Describe the problem**
Basically the spelling of starter code walkthrough is wrong . In on of the headings it is writtem stater code walkthrough

**Steps to reproduce?**
1. Go to...
2. Click on...
3. See error...
4. Change the spelling of stater to starter

**Versions**
_Android Studio version:_ 
_API version of the emulator:_ 

**Additional information**
_Include screenshots if they would be useful in clarifying the problem._

**Comment by osuleymanova** [comment]
@intSpartan 
Thank you for reaching out to us. The typo is fixed and will be published soon:

![Screen Shot 2022-07-13 at 4 05 16 PM](https://user-images.githubusercontent.com/86437155/178851789-4d7a4cf9-8a32-41a5-841e-30c505222a78.png)
